schema/mysql/v8/visibility/schema.sql

```diff
@@ -1,6 +1,7 @@
 CREATE TABLE executions_visibility (
   namespace_id            CHAR(64)      NOT NULL,
   run_id                  CHAR(64)      NOT NULL,
+  version                 BIGINT        NOT NULL DEFAULT 0,
```

Comment:
ditto: maybe put in a separate block and add comments explaining it

Reply:
So, its only an optimization, but generally fields which are read more will be more performant if earlier in column order.

Reply:
Considering that we will always read this on write, it seems worth doing. 

On the other hand, it seems like we read most columns, if not all.

Reply:
Hm, interesting. If that's the case, our order is not optimized at all. It should've been `namespace_id`, `workflow_id`, `run_id`, `version`, ...
Anyway, I don't think we're trying to optimize this much, but sure, not a big deal for me. But please, add some comments for the `version` column.

Reply:
Added comments